micromasters/settings.py

```diff
@@ -421,3 +422,12 @@ def get_var(name, default):
 
 # django-role-permissions
 ROLEPERMISSIONS_MODULE = 'roles.roles'
+
+CELERYBEAT_SCHEDULE = {
+    'batch-update-user-data-every-6-hrs': {
+        'task': 'dashboard.tasks.batch_update_user_data',
+        'schedule': crontab(minute=0, hour='*/6')
+    },
+}
+
+CELERY_TIMEZONE = 'UTC'
```

Comment:
I was talking about in the celery section in the settings! The settings must be grouped by topic.

Sent by my iDevice. May be rife with typos or poor judgment.

On Aug 2, 2016, at 8:11 AM, Amir Qayyum Khan <notifications@github.com<mailto:notifications@github.com>> wrote:


In micromasters/settings.py<https://github.com/mitodl/micromasters/pull/771#discussion_r73142901>:

> @@ -421,3 +422,12 @@ def get_var(name, default):
>
>  # django-role-permissions
>  ROLEPERMISSIONS_MODULE = 'roles.roles'
> +
> +CELERYBEAT_SCHEDULE = {
> +    'batch-update-user-data-every-6-hrs': {
> +        'task': 'dashboard.tasks.batch_update_user_data',
> +        'schedule': crontab(minute=0, hour='*/6')
> +    },
> +}
> +
> +CELERY_TIMEZONE = 'UTC'


then it wont work because it should be in settings
https://github.com/mitodl/micromasters/blob/master/micromasters/celery.py#L21

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/mitodl/micromasters/pull/771/files/7ed767b4cde4bc5aff469d3afa6d1e527596e517#r73142901>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AA9WlLHKiSY1C4cUfUSoebdy8XpCZfCqks5qbzPrgaJpZM4JZfY7>.